drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -39,6 +39,8 @@ def organizations(**args)
     end
 
     def client_search(input:, **args)
+      raise 'Invalid search. At least 1 search param is required.' unless input.to_h.excluding(:projects, :organizations).values.compact_blank.any?
```

Comment:
If this gets bubbled up to the UI, maybe we should make the language a bit less techy?   "Your search cannot be completed, at least one search term is required."

Reply:
`raise '..'` don't get bubbled up to the UI in production. In prod, users will see "an error occurred, it is being investigated, yadayada". I'm `raise`-ing here because it is an unexpected input-- the frontend application does not expose a way to pass no search params.

Reply:
Perfect then.  Thanks.